arm-trafficmanager/2017-03-01/swagger/trafficmanager.json

```diff
@@ -323,7 +323,7 @@
         "x-ms-pageable": {
           "nextLinkName": null
         },
-        "operationId": "Profiles_ListByInResourceGroup",
+        "operationId": "Profiles_ListAllInResourceGroup",
```

Comment:
I'd like to point our few things here:

1. Are you aware that changing the operation names is a breaking change in SDKs
2. Is this a expected change?
3. One of the SDK that would break is python [azure-mgmt-trafficmanager](https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-trafficmanager/azure/mgmt/trafficmanager/traffic_manager_management_client.py#L88)